AzureCore/Source/Util/CryptoUtils.swift

```diff
@@ -73,6 +53,27 @@ public enum HmacAlgorithm {
         return CCHmacAlgorithm(alg)
     }
 
+    /// Compute a hash of the underlying data using the specfied algorithm.
+    public func hash(_ data: UnsafeRawPointer!, _ len: CC_LONG, _ message: UnsafeMutablePointer<UInt8>!) -> Data {
```

Comment:
No, but there's no implementation of it in CommonCrypto, so we will have to do a manual implementation, and even that I couldn't track down easily. Ultimately, recognizing it will be non-trivial effort to support, I deferred it for future work. We'll have to decided whether we want to support it in Preview 1.